bkg/v2/BKG_v2.0.0.yaml

```diff
@@ -7068,7 +7005,7 @@ components:
           pattern: ^\S(?:.*\S)?$
           maxLength: 4
           description: |
-            Unique code for the different equipment size and type used to transport commodities. The code can refer to either the ISO size type (e.g. 22G1) or the ISO type group (e.g. 22GP) following the [ISO 6346](https://en.wikipedia.org/wiki/ISO_6346) standard.
+            Unique code for the different equipment size and type used to transport commodities. The code can refer to either the ISO size type (e.g. 22G1) or the ISO type group (e.g. 22GP) following the [ISO 6346:2022](https://www.iso.org/standard/83558.html) standard.
```

Comment:
By mentioning 2022, are we mentioning a particular version?

Reply:
Nice catch

Reply:
Good question - will revert...

Reply:
2022 removed - well spotted!!